Tasks/ChefKnife/task.json

```diff
@@ -7,8 +7,7 @@
     "category": "Deploy",
     "visibility": [
                 "Preview",
-                "Build",
-                "Release"
```

Comment:
I thought this is primarily for Release than build, and removing release seems changing it. If it is preview quality then isn't that good enough?